METRICS.md

```diff
@@ -108,6 +108,15 @@ An agent identifier should also be defined to identify which agent the metric is
 
     nifi.metrics.publisher.agent.identifier=Agent1
 
+### Configure Prometheus metrics publisher with SSL
+
+The communication between MiNiFi and the Prometheus server can be encrypted using SSL. This can be achieved by adding the SSL certificate path (a single file containing both the certificate and the SSL key) and optionally adding the root CA path when using a self signed certificate to the minifi.properties file. Here is an example with the SSL properties:
```

Comment:
The root CA is for the server certificate, isn't it?  I would make that clearer:
```suggestion
The communication between MiNiFi and the Prometheus server can be encrypted using SSL. This can be achieved by adding the SSL certificate path (a single file containing both the client certificate and the client SSL key) and optionally adding the root CA path if the Prometheus server uses a self-signed certificate, to the minifi.properties file. Here is an example with the SSL properties:
```

Reply:
In this scenario the MiNiFi publishes the metrics and acts as a server (using a CivetWeb server in the implementation) that can be scraped for the metrics by the Prometheus server. So in this case the MiNiFi needs a self signed certificate, not the Prometheus server.